datalad/resources/website/assets/js/main.js

```diff
@@ -78,6 +78,22 @@ function getParameterByName(name, url) {
 }
 
 /**
+ * Create Breadcrumbs to current location in dataset
+ * @return {array} html linkified breadcrumbs array
+ */
+function bread2crumbs() {
+  var raw_crumbs = loc().href.split('/');
+  var crumbs = [];
+  for (var index = 2; index < raw_crumbs.length() - 1; index++) {
+    if (raw_crumbs[index] !== '?dir=')
+      continue;
```

Comment:
This commit wasn't supposed to go out! I was in the middle of trying to create the breadcrumbs. Its supposed to be splitting the current loc by '/' and joining to form the href-ed breadcrumbs